File: c programs/Assignments/Assignment_3/Assign3_1.c
//Complete below code snippets it contains only service provider function. 
//Write entry point function to call below helper functions separately. 


//1.Write a program which accept one number from user and print that number of even numbers on screen. 

//Input : 7 
//Output: 2 4 6 8 10 12 14

#include<stdio.h>
void PrintEven(int iNo)
{
	int iCnt = 0;
	
	if(iNo <= 0)
	{
		return;
	}
	while(iCnt <= iNo)
	{
		if(iCnt % 2 == 0)
		{
			iNo = iNo + 1;
			printf("%d\t",iCnt);
		}
		iCnt++;
	}
}


int main()
{
	int iValue = 0;
	
	printf("enter the number:\n");
	scanf("%d",&iValue);
	
	PrintEven(iValue);
		
	return 0;
}

File: Python/Assignment_3/Assign3_1.py
#1.Write a program which accept N numbers from user and store it into List. Return addition of all
#elements from that List.

#Input : Number of elements : 6
#Input Elements : 13 5 45 7 4 56
#Output : 130 


def Add(LIST):
 iSum =0
 for i in range(len(LIST)):                      #range() sequence of no interation
   iSum = iSum + LIST[i]
 return iSum

def main():
 Arr=[]
 print("Enter elements:")
 iSize = int(input())
 
 for i in range(iSize):
   print("INPUT ELEMENT IS:",i+1)
   iValue=int(input())
   Arr.append(iValue)                           # Insert that element into List
   
   print("acept data",Arr)
   iRet = Add(Arr)
   
   print("Addition of elements:",iRet)
 
if __name__=="__main__":
  main()